source/slang/slang-emit-cpp.cpp

```diff
@@ -2629,130 +2666,7 @@ void CPPSourceEmitter::_emitForwardDeclarations(const List<EmitAction>& actions)
 
 void CPPSourceEmitter::emitModuleImpl(IRModule* module, DiagnosticSink* sink)
 {
-    // If we are emitting a heterogeneous program
```

Comment:
Have we lost some functionality - in terms of emitting GPU kernels as blobs? 

I understand the desire to want to remove the gfx specifics from an emit.

Also do we really have a 'heterogenous' example anymore?